src/Conversion/ONNXToKrnl/Tensor/ScatterElements.cpp

```diff
@@ -19,17 +19,6 @@ using namespace mlir;
 
 namespace onnx_mlir {
 
-// Returns true if all the indices are known to be positive and false otherwise.
-static bool indicesArePositiveConstants(Value indices) {
```

Comment:
Placed this utility function in ONNXtoKernlCommon.cpp so that it can be used by Gather, GatherElements, and ScatterElements during code generation (conversion from ONNX to Krnl).